SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -3390,10 +3445,275 @@ void CEulerSolver::GetPower_Properties(CGeometry *geometry, CConfig *config, uns
     delete [] Outlet_GrossThrust;
     delete [] Outlet_Force;
     delete [] Outlet_Power;
+    delete [] Outlet_DeltaP;
+    delete [] Outlet_Thrust;
+    delete [] Outlet_Torque;
+
+  }
+
+}
+
+// { actuatordisk-bem
+//.......function 'readsdata_' reads alpha, cl and cd values from the files........................
+//float readsdata_(char *sec_filename,propeller_geom_struct *s_prop, propeller_section_struct *sprop_sec)
+char* readsdata_(char *sec_filename,dpropeller_geom_struct *s_prop, dpropeller_section_struct *sprop_sec)
+{
+   int i,j,MAXRK,MAXALFK;
+   char dummy[1024];
+   FILE *file_ang;
+   int sect[BEM_MAXR];
+   char* fgets_r=0;
+//.Read blade section data from file
+   file_ang = fopen(sec_filename,"r");
+// To be read from file
+   fgets_r = fgets(dummy,120,file_ang);
+// Skip header
+   fgets_r = fgets(dummy,120,file_ang); sscanf(dummy,"%d", &s_prop->nblades);
+   fgets_r = fgets(dummy,120,file_ang); sscanf(dummy,"%lf", &s_prop->dia);
+   fgets_r = fgets(dummy,120,file_ang); sscanf(dummy,"%lf", &s_prop->rhub);
+   fgets_r = fgets(dummy,120,file_ang); sscanf(dummy,"%lf", &s_prop->ang0_75);
+//.Skip reading next line.................................................
+   fgets_r = fgets(dummy,120,file_ang);
+   fgets_r = fgets(dummy,120,file_ang);
+        sscanf(dummy,"%d%d",&MAXRK,&MAXALFK);
+   sprop_sec-> nrad = MAXRK;
+   sprop_sec-> nalf = MAXALFK;
+//
+   fgets_r = fgets(dummy,120,file_ang);
+   for(i=0; i < MAXRK; i++)
+   {
+      fgets_r = fgets(dummy,120,file_ang);
+      sscanf(dummy,"%d %lf %lf %lf",&sect[i],&sprop_sec->r1[i],&sprop_sec->chord[i],&sprop_sec->setangle[i]);
+   }
```

Comment:
I'm sorry but this has to be re-written in C++, please see if you can use the lookup table class in SU2 for this instead of a custom format.